traffic_server/tsb/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Fou:qndation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+FROM centos:7
+
+RUN	yum clean all \
+	&& yum install -y deltarpm epel-release centos-release-scl-rh \
+	&& yum-config-manager --enable rhel-server-rhscl-7-rpms \
+	&& yum clean all \
+	&& yum install -y \
+		autoconf \
+		automake \
+		pkgconfig \
+		libtool \
+		perl-ExtUtils-MakeMaker \
+		gcc-c++ \
+		glibc-devel \
+		tcl-devel \
+		expat-devel \
+		pcre \
+		pcre-devel \
+		libcap-devel \
+		flex \
+		hwloc-devel \
+		libuuid-devel \
+		lua-devel \
+		make \
+		git \
+		rpm-build \
+		man \
+		sudo \
+		python3 \
+		python35 \
+		ed \
+		zlib \
+		zlib-devel \
+		ncurses-devel \
+		libcurl-devel \
+		devtoolset-7 \
+		luajit-devel \
+		openssl \
+		openssl-devel \
+		hwloc \
+		perl-URI \
+		nmap-ncat \
+		perl-Digest-SHA \
+		nano \
+		ed \
+	&& yum clean all
+
+ADD	jansson.pic.patch /opt/src/
+ADD	cjose.pic.patch /opt/src/
+ADD	https://bootstrap.pypa.io/get-pip.py /
+RUN	python get-pip.py
+RUN	pip install --user Sphinx
+ADD	run.sh /
+ADD	trafficserver.spec /rpmbuilddir/SPECS/trafficserver.spec
+ADD	traffic_server_jemalloc /rpmbuilddir/SOURCES/traffic_server_jemalloc
+RUN	/usr/sbin/useradd -u 176 -r ats -s /sbin/nologin -d /
+CMD	set -o pipefail; scl enable devtoolset-7 ./run.sh 2>&1 | tee /rpmbuilddir/RPMS/x86_64/build.log
```

Comment:
Yeah, but we're not using debian. It works for this image and since the whole point of this image is building RPMs, it's unlikely that someone will swap the base image out. I'm leaving this one for now, I think.